features/download/index.html

```diff
@@ -35,6 +38,12 @@
        });
    });
 </script>
-  
+
+<script>
+   var blob = new Blob(["Hello world!"], {type: "application/octet-stream"});
+   var link = document.getElementById("blobDownloadLink");
```

Comment:
change those `var`s to `const`s and your can put "expert JavaScript" on your CV ;)